### PR TITLE
Feature/landing page

### PR DESCRIPTION
**Pull request by will-wow**
## Goals

Set up the new landing page

## Implementation

- Create a new `index.html` with the new landing page.
- Pull in a few of the base styles, but this is pretty different than the other pages, so mostly added custom styling in `landing.scss`.

## For discussion

- Where should the Learn More and Buy Tickets go? Should the Buy Tickets be disabled for now, or removed?
- For some reason the logo svgs exported by PhotoShop weren't showing up. I left the files for now so someone could take a look, and used pngs instead. I don't use PS much, so I probably just exported them incorrectly.

## Screenshots

empex-landing

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -53,6 +53,7 @@ GEM
     fastimage (1.8.1)
       addressable (~> 2.3, >= 2.3.5)
     ffi (1.9.10)
+    ffi (1.9.10-x64-mingw32)
```
Comment (will-wow) [inline Gemfile.lock]: I've got Photoshop on my windows machine, so I think this is just cruft from installing windows gems

File: config.rb [hunk 2]
```diff
@@ -28,12 +28,11 @@
 data.events.each do |year, events|
   events.each do |slug, event|
     proxy "events/#{year}/#{slug}.html", "event.html", locals: { event: event, year: year, slug: slug }, ignore: true
-    if event.current
-      proxy "index.html", "current_event.html", locals: { event: event, year: year, slug: slug }, ignore: true
-    end
   end
 end
 
+page 'index.html', layout: false
```
Comment (will-wow) [inline config.rb]: The landing page doesn't have any of normal layout (like the navbar)

File: source/assets/images/empex-logo-la-white.svg [hunk 3]
```diff
@@ -0,0 +1,31 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="240" height="249" viewBox="0 0 240 249">
```
Comment (will-wow) [inline source/assets/images/empex-logo-la-white.svg]: These svgs weren't displaying for me, I'm not sure why

**Comment by desmondmonster** [comment]
@will-wow `Learn More` should direct to each event page, either `/la` or `/nyc`.  `Buy Tickets` should be a disabled button that says "Tickets on Sale December 1"

File: config.rb (hunk 2 above)
Comment (desmondmonster) [reply config.rb]: see how awkward this whole CMS thing is?

File: source/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+  <meta name="twitter:card" content="summary" />
+  <meta name="twitter:site" content="@empexco" />
+  <meta name="twitter:title" content="EMPEX - An Elixir Movement." />
+  <meta name="twitter:description" content="EMPEX is a sophisticated conference series for the Elixir programming language and ecosystem held in New York City and Los Angeles."
+  />
+  <meta name="twitter:image" content="http://empex.co/assets/images/flag-577c45fc.jpg" />
+
+  <meta property="og:url" content="http://empex.co/" />
+  <meta property="og:title" content="EMPEX - Empire City Elixir Conference 2017" />
+  <meta property="og:description" content="EMPEX, the Empire City Elixir Conference, is a sophisticated conference series for the Elixir programming language and ecosystem held in New York City."
```
Comment (desmondmonster) [inline source/index.html.erb]: let's change these OG title and description tags to match the twitter tags above.

File: source/index.html.erb [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+<body class="landing">
+  <section id="top" class="top">
+    <div class="top__content">
+      <h1 class="top__name">EMPEX</h1>
+      <h2 class="top__subhead">An Elixir Movement.</h2>
+
+      <p class="top__description">
+        One-day conferences for curious programmers.<br/>
+        Coming to New York City and Los Angeles.
```
Comment (desmondmonster) [inline source/index.html.erb]: _Happening_ in New York City and Los Angeles

**Comment by desmondmonster** [comment]
I'm just going to add another commit with some minor copy changes

**Comment by will-wow** [comment]
@desmondmonster I pushed up the tweaks from your comments (except for getting the buttons to look disabled). But feel free to commit away!

**Comment by desmondmonster** [comment]
@will-wow ok.  how's the progress on the LA event page?

**Comment by will-wow** [comment]
@desmondmonster I just pushed a commit to have the NYC card link back to the old homepage - does that seems like a reasonable default for now?

As for the LA page - I talked to Rosemary today, and she was working on it. I'll check in on that tomorrow, but we could always deploy this with a disabled "coming soon" button for the More Info button. In the meantime, it sounds like this PR is done, so I can at least merge it into the `develop` branch and give Rosemary something to rebase off of

**Pull request merged**